src/configurations/destinations/tiktok_ads/schema.json

```
@@ -831,7 +831,53 @@
               ]
             }
           },
+          "shopify": {
```

Biome (1.9.4)

[error] The key shopify was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)